src/data/html-quiz.ts

```diff
@@ -1,4 +1,16 @@
 const htmlQuiz = [
+  {
+    Question:
+    "Which of the following is the correct way of creating an hyperlink in HTML?",
+    Answer: "<a href= “www.freecodecamp.org”> freeCodeCamp </a>",
+    Distractor1: "<a>www.freecodecamp.org <freeCodeCamp /a>",
+    Distractor2: "<a href=“ www.freecodecamp.org” freeCodeCamp /a>",
+    Distractor3: "<a link=“ www.freecodecamp.org” freeCodeCamp > </a>",
+    Explanation:
+      "In HTML a href tag is used to hyperlink text as shown in the given syntax. <a href= “www.link_name.com”> hyperlinked_text </a>"
+    Link:
+      "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-href"
```

Comment:
Whenever possible, it is always best to use a freeCodeCamp resources first

```suggestion
      "https://www.freecodecamp.org/news/the-html-a-tag-anchor-tag-example-code/"
```